executor/src/witgen/machines/machine_extractor.rs

```diff
@@ -186,16 +229,41 @@ pub fn split_out_machines<'a, T: FieldElement>(
     // Note that this machine comes last, because some machines do a fixed lookup
     // in their take_witness_col_values() implementation.
     // TODO: We should also split this up and have several instances instead.
+
+    // Compute sizes of fixed lookup multiplicity columns.
```

Comment:
Is it possible to move the creation of the fixed lookup machine to a helper function?

Reply:
Moved to `FixedLookup::new`